src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Select } from '../../types/otherTypes';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface TodoModalProps {
+  selectedTodo: Todo;
+  onSelect: (todo: Select) => void;
+}
+
+const defaultUserData = {
+  id: 0,
+  name: '',
+  email: '',
+  phone: '',
+};
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  selectedTodo,
+  onSelect,
+}) => {
+  const [user, setUser] = useState<User>(defaultUserData);
```

Comment:
```suggestion
  const [user, setUser] = useState<User | null>(null);
```
just set null by default